examples/darts/rnn/constant.yaml

```diff
@@ -0,0 +1,67 @@
+description: darts_rnn_nas
+
+data:
+  data_download_dir: /tmp 
```

Comment:
For consistency, this `data_download_dir` is `tmp`, but dart_cnn has `data`. I prefer maintaining consistency with the latter if that's not too much of a code change